x-pack/plugins/fleet/common/types/models/epm.ts

```diff
@@ -72,6 +72,7 @@ export enum KibanaAssetType {
   map = 'map',
   lens = 'lens',
   securityRule = 'security_rule',
+  cloudSecurityPostureRuleTemplate = 'cloud_security_posture_rule_template',
```

Comment:
this breaks the package-spec definition. lets change the folder name to be `csp_rule_template` if possible and deal with the renaming later

Reply:
also let's try and not mix old naming and new naming please.
Here is the old [PR](https://github.com/elastic/kibana/pull/125821) from Uri.